cpp/ps0001/p0056.so1.cpp

```diff
@@ -4,13 +4,14 @@ using namespace std;
 
 int main()
 {
-
-    int x , y , o ;
-
+    // prompting user for 3 numbers to add 1/2
+    unsigned short num1, num2, num3;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in the right way to prompt user for the 3 numbers